### ISSUE TITLE
Fine-tune for downstream regression task (continuous labels)

### ISSUE THREAD

**Issue by Fragola7** [rmaphoh/RETFound_MAE#13]
Hi, thanks for your excellent work!

In the paper, you provide examples where the model is fine-tuned for downstream classification tasks (either binary diseases or multi-class labels). Is it possible to use the same framework but to fine-tune on continuous labels, for example predicting systolic blood pressure from fundus images? If so, could you please provide guidelines on how this should be done?

**Comment by rmaphoh** [comment]
Thanks. This can be done by defining a prediction head in model_vit.py. The loss function should be revised accordingly.

I currently have no bandwidth to add this function, but probably will do it in the coming months

**Comment by Fragola7** [comment]
Thanks for your answer! I will look into it.
